KishanJ-Patel/One-Stop-Shopper
# Change the product list layout to vertically scroll on home page

### Context

- This pull request addresses the issue OSS-34 which describes the bug where the product list layout is horizontally scrolled.
- The bug makes it difficult for the users to view the products.

### Changes

- Replaced grid layout with flex layout for the div containing the product list.
- Create a new property for the product card component to apply tailwind CSS classes to the component.
- Changed the product list layout to vertically scroll.

### Testing

- To test this bugfix:
  1. Open the home page of the web app.
  2. Observe the layout of the product list.

### Related Issues

- Closes `OSS-34`

### Reviewer Requests

- @KishanJ-Patel : For frontend review
- @KishanJ-Patel : For backend review
- @KishanJ-Patel : For testing and QA
